moorhuhn-service/src/main/java/com/moorhuhnservice/moorhuhnservice/BaseClasses/Question.java

```diff
@@ -0,0 +1,96 @@
+package com.moorhuhnservice.moorhuhnservice.BaseClasses;
+
+
+import javax.persistence.*;
+@Entity
+@Table(name = "questions")
+public class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;

```

Comment:
```suggestion
    private long id;
```
The `id` is definitely not optional.

Reply:
where is this optional?

Reply:
In it being a Wrapper at the moment.
That's the reason for defining your server side model on wrappers:
To signify that they are not required.